packages/smooth_app/lib/pages/scan/mkit_scan_helper.dart

```diff
@@ -150,6 +164,12 @@ class _MLKitScanDecoderMainIsolate {
     _isolate?.kill(priority: Isolate.immediate);
     _isolate = null;
   }
+
+  bool get isDisposed =>
+      _isIsolateInitialized == false &&
+          _completer == null &&
+          _sendPort == null ||
+      _isolate == null;
```

Comment:
Could we add some parenthesis for clarity?

Reply:
Oops this should be an &&.

Reply:
ok, waiting for the fix @g123k